sys-process/pm2/pm2-5.3.0.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit nodejs-mod systemd
+
+DESCRIPTION="Process manager for Node.js applications with a built-in load balancer"
+HOMEPAGE="https://pm2.keymetrics.io/"
+SRC_URI="https://github.com/Unitech/pm2/archive/${PV}.tar.gz -> ${P}.tar.gz
+			https://raw.githubusercontent.com/inode64/inode64-overlay/main/dist/${P}-node_modules.tar.xz"
```

Comment:
We should really have an approach like dotnet where we allow downloading them too, I think, unless the list is too big.

Reply:
Many packages that I consider candidates for this kind of inclusion actually have few dependencies. Few meaning less than 50.